.circleci/continue_config.yml

```diff
@@ -0,0 +1,771 @@
+                          "text": {
+                            "type": "plain_text",
+                            "text": "View Job"
+                          },
+                          "url": "${CIRCLE_BUILD_URL}"
+                        }
+                      ]
+                    }
+                  ],
+                  "color": "good"
+                }
+              ]
+            }
+
+executors:
+  node:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:14.18.1-browsers # For latest available images check – https://circleci.com/docs/2.0/docker-image-tags.json
+
+jobs:
+
+  working_directory: ~/repo
+
+  install:
+    executor: node
+    environment:
+      # required to prevent ENOMEM errors
+      LERNA_ARGS: --concurrency 1
+    steps:
+      - checkout
+      # Restore 'master' cache
+      - restore_cache_dist_directories
+      - install_node_dependencies
+      - persist_to_workspace:
+          root: .
+          paths:
+            - '*'
+      - slack_notify_fail
+      - slack_notify_success
+
+  danger-checks:
+    executor: node
+    steps:
+      - attach_workspace:
+          at: .
+      - run: # Run PR Checks
+         name: Run PR Checks
+         command: yarn danger ci --failOnErrors
+      - slack_notify_fail
+      - slack_notify_success
+
+  build:
+    executor: node
+    environment:
+      # required to prevent ENOMEM errors
+      LERNA_ARGS: --concurrency 1
+    parameters:
+      component_type:
+        type: string
+        # Setting a default value so this job can be used by both workflows. Value will always be overridden at workflow level.
+        default: global
+      component_type_folder:
+        type: string
+        # Setting a default value so this job can be used by both workflows. Value will always be overridden at workflow level.
+        default: global
+    steps:
+      - attach_workspace:
+          at: .
+        # If global-change - Build + Lint all packages.
+      - when:
+          condition: << pipeline.parameters.global-change >>
+          steps:
+            # Build components based on atomic level - e.g ci:build:atoms
+            - build_component_type:
+                component_type: << parameters.component_type >>
+            - lint_component_type:
+                component_type: << parameters.component_type >>
+            - persist_to_workspace:
+                root: << parameters.component_type_folder >>
+                paths:
+                  - '*'
+            #Only save cache dist if master branch
+            - when:
+                condition:
+                  equal: [ master, << pipeline.git.branch >> ]
+                steps:
+                  - save_cache_dist_directories
+      # Else Build + Lint changed packages.
+      - when:
+          condition:
+            not: << pipeline.parameters.global-change >>
+          steps:
+            # Build all components
+            - build_all_components
```

Comment:
The naming here is probably quite confusing. I'll try and rename to something better.

Reply:
`build_all_packages` or `build_everything` perhaps?